cis194/week1/zhansongl/credit_card.hs

```diff
@@ -0,0 +1,125 @@
+import Test.QuickCheck
+import Test.Hspec
+
+toDigitsRev :: Integer -> [Integer]
+toDigitsRev n
+  | n >= 0 = digit n
+  | otherwise = []
+  where digit 0 = []
+        digit n = n `mod` 10 : digit (n `div` 10)
```

Comment:
For positive integer, rem is preferred for performance reason.

Reply:
hmm, didn't know about this, will fix